src/toxcore/state_format/old.rs

```diff
@@ -346,14 +349,14 @@ pub struct FriendState {
     friend_status: FriendStatus,
     pk: PublicKey,
     /// Friend request message that is being sent to friend.
-    fr_msg: Vec<u8>,
+    pub fr_msg: Vec<u8>,
```

Comment:
Why did you change from private to public?

Reply:
done